pkg/dogstatsd/server.go

```diff
@@ -163,6 +163,16 @@ func NewServer(aggregator *aggregator.BufferedAggregator) (*Server, error) {
 		}
 	}
 
+	pipeName := config.Datadog.GetString("dogstatsd_windows_pipe_name")
+	if len(pipeName) > 0 {
+		namedPipeListener, err := listeners.NewNamedPipeListener(pipeName, packetsChannel, sharedPacketPool)
+		if err != nil {
+			log.Errorf(err.Error())
```

Comment:
I think when we error out we'll get an underlying lib error from `winio` (for example) but it might be immediately obvious in the logs that it's from the windows named pipe, I would maybe add something like:
```
log.Errorf("named pipe error: %v", err.Error())
```